app/models/services/service_key.rb

```diff
@@ -0,0 +1,95 @@
+module VCAP::CloudController
+  class ServiceKey < Sequel::Model
+    class InvalidAppAndServiceRelation < StandardError; end
+
+    many_to_one :service_instance
+
+    export_attributes :name, :service_instance_guid, :credentials, :syslog_drain_url
+
+    import_attributes :name, :service_instance_guid, :credentials, :syslog_drain_url
+
+    alias_attribute :broker_provided_id, :gateway_name
```

Comment:
This has no tests. Luckily, this is a relic from the days of V1 service gateways and we don't need it anymore. I'm adding a chore to investigate removing this for service_bindings and service_instances.